Makefile

```diff
@@ -2,5 +2,10 @@ PANDOCFLAGS = \
 	      --table-of-contents \
 	      --number-sections
 
-build/book.pdf: book.md Makefile
+all: build/book.pdf
```

Comment:
Perhaps we should also have:
.PHONY: all
?

Reply:
That makes sense, thank you!
I've added the .PHONY line just now in the latest update.